Error 2021-04-18 23:02:38: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file 'vscode-remote://wsl+ubuntu-20.04/home/jdreeder1/.config/Code/User/settings.json' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open '/home/jdreeder1/.config/Code/User/settings.json')
	at _handleError (/home/jdreeder1/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:769:473)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async m.readFile (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:49:864350)
	at async g.fixSettingsFile (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:47:290713)
	at async Promise.all (index 0)
	at async g.updateSettings (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:47:292433)] {
  code: 'FileNotFound',
  name: 'EntryNotFound (FileSystemError)'
}
Error 2021-04-18 23:02:38: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): [EntryNotFound (FileSystemError): Unable to read file 'vscode-remote://wsl+ubuntu-20.04/home/jdreeder1/.vscode/settings.json' (EntryNotFound (FileSystemError): Error: ENOENT: no such file or directory, open '/home/jdreeder1/.vscode/settings.json')
	at _handleError (/home/jdreeder1/.vscode-server/bin/ea3859d4ba2f3e577a159bc91e3074c5d85c0523/out/vs/server/remoteExtensionHostProcess.js:769:473)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
	at async m.readFile (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:49:864350)
	at async g.fixSettingsFile (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:47:290713)
	at async Promise.all (index 1)
	at async g.updateSettings (/home/jdreeder1/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/out/client/extension.js:47:292433)] {
  code: 'FileNotFound',
  name: 'EntryNotFound (FileSystemError)'
}
> ~/project_venv/bin/python3 ~/.vscode-server/extensions/ms-toolsai.jupyter-2021.2.576440691/pythonFiles/pyvsc-run-isolated.py pip list
